packages/esm-patient-attachments-app/src/attachments/attachments-overview.component.tsx

```diff
@@ -98,7 +98,12 @@ const AttachmentsOverview: React.FC<AttachmentsOverviewProps> = ({ patientUuid }
 
   const showAddAttachmentModal = useCallback(() => {
     const close = showModal('capture-photo-modal', {
-      saveFile: (file: UploadedFile) => createAttachment(patientUuid, file),
+      saveFile: (file: FileAttachment) => {
+        if (file.capturedFromWebcam && !file.fileName.includes('.')) {
+          file.fileName = `${file.fileName}.png`;
+        }
+        return createAttachment(patientUuid, file);
+      },
```

Comment:
This is the key change. If the file is captured from the webcam and lacks a file extension, append the `png` extension to it by default.

Reply:
What leads to the cases where the webcam file extension does not exist?

Reply:
Basically, the user specifies the file name. If they don't include an extension, then there's no extension.

Reply:
oh i see!!!